input/tex/extensions/upgreek.rst

```diff
@@ -0,0 +1,41 @@
+.. _tex-upgreek:
+
+#######
+upgreek
+#######
+
+The `upgreek` extension implements the ``upgreek`` style package from LaTeX. It
+provides upright Greek characters for both lower and upper cases.  See the `CTAN
+page <https://www.ctan.org/pkg/upgreek>`__ for more information and
+documentation of `upgreek`.
+
+Note, that the extension does not implement the font selection options from the
+LaTeX package.
```

Comment:
Missing "This package isn't autoloaded" message.

Reply:
Added.